test/SBP_operators_test.jl

```diff
@@ -8,7 +8,8 @@ D_test_list = (MattssonNordström2004(), MattssonSvärdNordström2004(),
                 MattssonAlmquistCarpenter2014Optimal(),
                 MattssonAlmquistVanDerWeide2018Minimal(),
                 MattssonAlmquistVanDerWeide2018Accurate(),
-                DienerDorbandSchnetterTiglio2007())
+                DienerDorbandSchnetterTiglio2007(), 
+                SharanBradyLivescu2022(1.0, 1.0))
```

Comment:
> Three higher derivative tests are currently failing locally due to roundoff issues. Since there's a parameter `alpha::Float64` involved in the formulas, I am not sure that it is quite as accurate as the other operators.
> 
> Any suggestions?

Tests seem to look good in CI. Did you try using rationals, e.g., `alpha_left = alpha_right = 1//1` here

Reply:
Good idea. Let me give that a shot locally.

Reply:
It doesn't seem to make much of a difference. My guess is that the formulas would need to be computed in higher precision to avoid roundoff in computing the third derivative, but this might get expensive.

Reply:
> computing the third derivative

Didn't you implement only first-derivative operators here? And didn't they derive only first- and second-derivative operators in their paper?

Could you maybe post the code that fails locally for you, please?

Reply:
Interesting. The tests that fail for me fall inside the third derivative test block https://github.com/ranocha/SummationByPartsOperators.jl/blob/c19055576084fecb6a49bfe402e6bbdacd4f4cfc/test/SBP_operators_test.jl#L469 or the fourth derivative test block. Specifically, the lines that fail tests are 

https://github.com/ranocha/SummationByPartsOperators.jl/blob/c19055576084fecb6a49bfe402e6bbdacd4f4cfc/test/SBP_operators_test.jl#L642

https://github.com/ranocha/SummationByPartsOperators.jl/blob/c19055576084fecb6a49bfe402e6bbdacd4f4cfc/test/SBP_operators_test.jl#L836

and 

https://github.com/ranocha/SummationByPartsOperators.jl/blob/c19055576084fecb6a49bfe402e6bbdacd4f4cfc/test/SBP_operators_test.jl#L838

Reply:
Here is the stacktrace for the first failed test in case it helps

```julia
     Testing Running tests...
Non-Periodic Operators: Test Failed at /Users/jchan985/.julia/dev/SummationByPartsOperators/test/SBP_operators_test.jl:642
  Expression: isapprox(derivative_right(D, x2, Val{2}()), 2, atol = 10000 * eps(T))
   Evaluated: isapprox(2.002176f0, 2; atol = 0.0011920929f0)

Stacktrace:
  [1] macro expansion
    @ /Applications/Julia-1.10.app/Contents/Resources/julia/share/julia/stdlib/v1.10/Test/src/Test.jl:672 [inlined]
  [2] top-level scope
    @ ~/.julia/dev/SummationByPartsOperators/test/SBP_operators_test.jl:642
  [3] include
    @ ./client.jl:489 [inlined]
  [4] macro expansion
    @ ~/.julia/dev/SummationByPartsOperators/test/runtests.jl:8 [inlined]
  [5] macro expansion
    @ /Applications/Julia-1.10.app/Contents/Resources/julia/share/julia/stdlib/v1.10/Test/src/Test.jl:1577 [inlined]
  [6] macro expansion
    @ ~/.julia/dev/SummationByPartsOperators/test/runtests.jl:8 [inlined]
  [7] macro expansion
    @ ./timing.jl:275 [inlined]
  [8] macro expansion
    @ ~/.julia/dev/SummationByPartsOperators/test/runtests.jl:8 [inlined]
  [9] macro expansion
    @ ./timing.jl:275 [inlined]
 [10] macro expansion
    @ ~/.julia/dev/SummationByPartsOperators/test/runtests.jl:6 [inlined]
 [11] macro expansion
    @ /Applications/Julia-1.10.app/Contents/Resources/julia/share/julia/stdlib/v1.10/Test/src/Test.jl:1577 [inlined]
 [12] macro expansion
    @ ~/.julia/dev/SummationByPartsOperators/test/runtests.jl:6 [inlined]
 [13] macro expansion
    @ ./timing.jl:275 [inlined]
 [14] top-level scope
    @ ~/.julia/dev/SummationByPartsOperators/test/runtests.jl:265
```

Reply:
Reran with v1.9.2 and everything passes; it appears to be a v1.10 issue. @ranocha are you OK with resolving this for now and re-examining when the v1.10 beta comes out?

Reply:
Yes, that sounds good to me. Thanks for reporting!